.github/workflows/dart-tests.yaml

```
@@ -72,7 +72,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        flutter-version: ['3.19.0', '3.29.0']
+        flutter-version: ['3.24.5', '3.29.0']
```

```suggestion
        flutter-version: ['3.24.5', '3.32.1']
```
perhaps? or `latest`?

---

Only mentioning it here, but we should be consistent if we do upgrade.

---

Decision:
- Agreed that we should update the maximum version to the latest Flutter stable release, [3.32.1](https://github.com/flutter/flutter/releases/tag/3.32.1)